dialog/lib/dialog.ts

```diff
@@ -26,7 +26,7 @@ const CLOSING_TRANSITION_PROP = '--_closing-transition-duration';
 /**
  * Dialog transition type.
  */
-export type DialogTransition = 'grow'|'shrink'|'grow-down'|'grow-up'|'grow-left'|'grow-right';
+export type DialogTransition = 'grow'|'shrink'|'grow-down'|'grow-up'|'grow-left'|'grow-right'|'';
```

Comment:
Also I thought good to add the empty option to the list, that is also one value in the knob select drop down.